docs/src/FAQ.md

```diff
@@ -76,3 +76,105 @@ ChainRulesTestUtils.jl has some dependencies, so it is a separate package from C
+\end{pmatrix}
+
+```
+and `rrule` corresponds to
+```math
+\begin{pmatrix}
+\Delta u \, \tfrac{\partial u}{\partial x} + \Delta v \, \tfrac{\partial v}{\partial x}
+&
+\Delta u \, \tfrac{\partial u}{\partial y} + \Delta v \, \tfrac{\partial v}{\partial y}
+\end{pmatrix}
+=
+\begin{pmatrix}
+\Delta u & \Delta v
+\end{pmatrix}
+\begin{pmatrix}
+\tfrac{\partial u}{\partial x} & \tfrac{\partial u}{\partial y} \\
+\tfrac{\partial v}{\partial x} & \tfrac{\partial v}{\partial y} \\
+\end{pmatrix}
+.
+```
+The Jacobian of ``f:\mathbb{C} \to \mathbb{C}`` interpreted as a function ``\mathbb{R}^2 \to \mathbb{R}^2`` can hence be evaluated using either of the following functions.
+```
+function jacobian_via_frule(f,z)
+    fz,df_dx = frule((Zero(), 1),f,z)
+    fz,df_dy = frule((Zero(),im),f,z)
+    return [
+        real(df_dx)  real(df_dy)
+        imag(df_dx)  imag(df_dy)
+    ]
+end
+```
+```
+function jacobian_via_rrule(f,z)
+    fz, pullback = rrule(f,z)
+    _,du_dz = pullback( 1)
+    _,dv_dz = pullback(im)
+    return [
+        real(du_dz)  imag(du_dz)
+        real(dv_dz)  imag(dv_dz)
+    ]
+end
+```
+
+If ``f(z)`` is holomorphic, then the derivative part of `frule` can be implemented as ``f'(z) \, \Delta z`` and the derivative part of `rrule` can be implemented as ``\Delta f \,  \overline{f'(z)}``.
+Consequently, holomorphic derivatives can be evaluated using either of the following functions.
+```
+function holomorphic_derivative_via_frule(f,z)
+    fz,df_dz = frule((Zero(),1),f,z)
+    return df_dz
+end
+```
+```
+function holomorphic_derivative_via_rrule(f,z)
+    fz, pullback = rrule(f,z)
+    dself, conj_df_dz = pullback(1)
+    return conj(conj_df_dz)
+end
+```
+
+In the multi-dimensional case ``f : \mathbb{C}^m \to \mathbb{C}^n``, the derivative part of `frule` is given by
+```math
+\sum_{\ell = 1}^m \tfrac{\partial u_k}{\partial x_\ell} \, \Delta x_\ell + \tfrac{\partial u_k}{\partial y_\ell} \, \Delta y_\ell + i \, \Bigl( \tfrac{\partial v_k}{\partial x_\ell} \, \Delta x_\ell + \tfrac{\partial v_k}{\partial y_\ell} \, \Delta y_\ell \Bigr)
+\qquad\text{for } k = 1, \ldots, n
+```
+and the derivative part of `rrule` is given by
+```math
+\sum_{k = 1}^n \Delta u_k \, \tfrac{\partial u_k}{\partial x_\ell} + \Delta v_k \, \tfrac{\partial v_k}{\partial x_\ell} + i \, \Bigl(\Delta u_k \, \tfrac{\partial u_k }{\partial y_\ell} + \Delta v_k \, \tfrac{\partial v_k}{\partial y_\ell} \Bigr)
+\qquad\text{for } \ell = 1, \ldots, m
+```
+In particular, if ``f(z)`` is holomorphic (i.e. every component function ``f_k(z)`` is holomorphic in every component ``z_\ell`` of the argument), then the derivative part of `frule` can be implemented as ``\nabla f(z) \, \Delta z`` and the derivative part of `rrule` can be implemented as ``\overline{\nabla f(z)}^H \, \Delta f``, where ``\nabla f(z)`` denotes the matrix of holomorphic derivatives ``(\nabla f)_{k\ell} = \frac{\partial f_k}{\partial z_\ell}`` and ``A^H = \overline{A}^T`` denotes the Hermitian transpose.
```

Comment:
In `\overline{\nabla f(z)}^H`, does the overline indicate conjugation, and if so, isn't it then redundant?

Reply:
Indeed, the extra `\overline` was wrong. Fixed now.